packages/console/src/pages/Connectors/components/ConnectorRow/index.tsx

```diff
@@ -67,14 +68,15 @@ const ConnectorRow = ({ type, connectors, onClickSetup }: Props) => {
       </td>
       <td>{connectorTypeColumn}</td>
       <td>
-        {inUse !== undefined && (
-          <Status status={inUse ? 'enabled' : 'disabled'}>
-            {t('connectors.connector_status', {
-              context: inUse ? 'in_use' : 'not_in_use',
-            })}
-          </Status>
-        )}
-        {inUse === undefined && !connectors[0] && '-'}
+        {conditional(
+          firstConnector && (
+            <Status status={inUse ? 'enabled' : 'disabled'}>
+              {t('connectors.connector_status', {
+                context: inUse ? 'in_use' : 'not_in_use',
+              })}
+            </Status>
```

Comment:
Can we just pass the `isInUse` as a prop into this `<Status />` component and let it handle the logic inside?

Reply:
The status is a common component, and it's not related to the connector.

Reply:
ok